.env.example

```diff
@@ -33,4 +33,49 @@ PUBLIC_ASSETS_URL=http://localhost:3000/assets
 # LiveKit (voice chat)
 LIVEKIT_WS_URL=
 LIVEKIT_API_KEY=
-LIVEKIT_API_SECRET=
\ No newline at end of file
+LIVEKIT_API_SECRET=
+
+# ==============================================
+# DATABASE CONFIGURATION
+# ==============================================
+# Configure your database connection here.
+# 
+# Supported databases:
+# - SQLite (default - no configuration needed)
+# - PostgreSQL (DB_TYPE=pg with DB_URL)
+#
+# If DB_TYPE and DB_URL are not provided or empty,
+# the application will automatically use SQLite in the world folder.
+
+# Database type - leave empty for SQLite, or 'pg' for PostgreSQL
+DB_TYPE=
+
+# Database connection URL - leave empty for SQLite
+# For PostgreSQL, use format: postgres://username:password@host:port/database?schema=public
+DB_URL=
+
+# ==============================================
+# STORAGE CONFIGURATION
+# ==============================================
+# Configure your file storage system here.
+# 
+# Supported storage types:
+# - Local file system (default - no configuration needed)
+# - S3-compatible storage (CloudFlare R2, AWS S3, DigitalOcean Spaces, etc.)
+#
+# If STORAGE_TYPE is set to 's3', S3-compatible storage will be used.
+# If STORAGE_TYPE is empty or not set, local file storage will be used.
+# 
+# S3 credentials can be provided via environment variables below,
+# or through AWS CLI configuration, IAM roles, or instance profiles.
+
+# S3 storage configuration (only required when using S3)
+# To use S3-compatible storage, set STORAGE_TYPE and other S3 variables below
+STORAGE_TYPE=
+S3_BUCKET_NAME=bucket-name
+S3_ACCESS_KEY_ID=access-key-id
+S3_SECRET_ACCESS_KEY=secret-key-id
+S3_REGION=eu-west-1
+S3_ASSETS_PREFIX=
+S3_COLLECTIONS_PREFIX=
+S3_STORAGE_PREFIX=
```

Comment:
can we do uri's for S3 too?

Reply:
Implemented S3 URIs as suggested! 

Now you can use STORAGE_URL=s3://bucket-name/folder-path/assets/ with standard AWS credentials, (just like DB_URL approach).